### ISSUE TITLE
Discrepancy between Paper and Official Code: SpaceLLaVA vs SpaceMantis

### ISSUE THREAD

**Issue by gompanghee** [chanhee-luke/RoboSpatial-Eval#5]
## Summary

  The paper states that **SpaceLLaVA** was used for evaluation, but the official evaluation code uses **SpaceMantis**. These are completely different models, and the paper should be corrected to reflect the actual model used.

  ## Evidence

  ### Paper (Appendix C.2):
  > "We initialize the model from remyxai/SpaceLLaVA from Hugging Face"

  ### Official Code (models.py, line 41):
  ```python
  def load_spatialvlm_model(model_path=None):
      if model_path is None:
          model_path = "remyxai/SpaceMantis"  # NOT SpaceLLaVA
 ```

  SpaceLLaVA vs SpaceMantis

  These are completely different models:

  | Attribute      | SpaceLLaVA     | SpaceMantis             |
  |----------------|----------------|-------------------------|
  | Base Model     | LLaVA-v1.5-13B | Mantis-8B-siglip-llama3 |
  | Parameters     | 13.4B          | 8B                      |
  | Training Data  | 28k samples    | 185k samples            |
  | Vision Encoder | CLIP           | SigLIP                  |
  | Language Model | Vicuna 1.5     | Llama 3                 |

  ## Reproduction Results

  I tested both SpaceLLaVA-13B and SpaceMantis on RoboSpatial-Home:

  | Category      | SpaceLLaVA in Paper (Table 4) | SpaceLLaVA-13B | SpaceMantis |
  |---------------|-----------------|----------------|-------------|
  | Configuration | 61.0%           | 67.5%          | 61.0% ✓     |
  | Context       | 2.5%            | 0.0%           | 2.5% ✓      |
  | Compatibility | 61.0%           | 76.2%          | 42.9%       |

  - SpaceMantis matches the paper's Configuration (61.0%) and Context (2.5%) results exactly
  - SpaceLLaVA-13B produces completely different results across all categories
  - SpaceLLaVA-13B outputs bounding boxes instead of points for Context questions, resulting in 0% accuracy

  This strongly indicates that SpaceMantis was the model actually used in the paper's experiments, not SpaceLLaVA.

  ## Questions

  1. Was the paper evaluated using SpaceMantis, not SpaceLLaVA?
  2. There is an 18% discrepancy in the Compatibility results (61.0% in paper vs 42.9% reproduced with SpaceMantis). I tested both the current and initial versions of SpaceMantis (Aug 2024) and obtained the same 42.9% result. Could you clarify how the 61.0% Compatibility score was obtained?

**Comment by chanhee-luke** [comment]
After further investigation, we found that the compatibility for SpaceLLava-13B is 41.0, similar to what you reported. This means that the improvement from RoboSpatial is much larger than what we expected! Thanks for pointing it out, this will be updated in our manuscript.
